07-routing-gossip.md

```diff
@@ -497,14 +497,18 @@ The receiving node:
   - if the specified `chain_hash` value is unknown (meaning it isn't active on
   the specified chain):
     - MUST ignore the channel update.
-  - if `timestamp` is NOT greater than that of the last-received
+  - if the `timestamp` is equal to the last-received `channel_update` for this
+    `short_channel_id` AND `node_id`:
+  	- if the fields below `timestamp` differ:
+		- MAY blacklist this `node_id`.
+		- MAY forget all channels associated with it.
+	- if the fields below `timestamp` are equal:
+		- SHOULD ignore this message	
+  - if `timestamp` is lower than that of the last-received
   `channel_update` for this `short_channel_id` AND for `node_id`:
     - SHOULD ignore the message.
   - otherwise:
```

Comment:
I guess this otherwise could have been removed too?

Reply:
@jtimon No that otherwise is needed. Just that the `if` and `otherwise` should be indented on the previous otherwise. I think there was a formatting error when creating the pull request.

Reply:
Is reverting the commit and making changes the right way to go?

Reply:
No we don't rewrite history. You can submit a tiny indentation fix PR and it should be approved very quickly.

Reply:
@t-bast cool. Will submit one soon.

Reply:
Thanks!